packages/core-utils/src/common/metrics.ts

```diff
@@ -0,0 +1,31 @@
+import prometheus, {
+  collectDefaultMetrics,
+  DefaultMetricsCollectorConfiguration,
+  Registry,
+} from 'prom-client'
+
+export interface MetricsOptions {
+  prefix: string
+  labels?: Object
```

Comment:
Should this have a more specific type?

Reply:
prom-client makes it so that it's generic: https://github.com/siimon/prom-client/blame/master/README.md#L83-L91

i'm imagining adding environment as a label but not sure yet, can make an issue to make this type specific as we finalize labels, or remove until we start using labels - what do you think?